portal/templates/index.njk

```diff
@@ -33,6 +33,17 @@
 
     <div class="govuk-width-container">
 
+      {% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+      {% if backLink and backLink.href %}
```

Comment:
Should we also make the same change in the `gef-ui` package?

Reply:
Discussed with @ChristianMcc : we would need to run this change past dtfs and the back link is apparently working in gef-ui, so let's not also change gef-ui